tutorials/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -12,14 +12,14 @@ and (re)visit the :ref:`Scripting section <doc_scripting>` of the
 step-by-step tutorial.
 
 C# is a high-level programming language developed by Microsoft. In Godot,
-it is implemented with .NET 8.0.
+it is implemented with .NET 9.0.
 
 .. attention::
 
     Projects written in C# using Godot 4 currently cannot be exported to the web
     platform. To use C# on the web platform, consider Godot 3 instead.
-    Android and iOS platform support is available as of Godot 4.2, but is
-    experimental and :ref:`some limitations apply <doc_c_sharp_platforms>`.
```

Comment:
Our Android and iOS support is still considered experimental, and we add a warning on the export dialog accordingly.

We'll likely remove the experimental status for Android on 4.6 if everything goes well, but I don't think we'll do that for iOS because it relies on NativeAOT and we don't have good trimming support which is a prerequisite.